examples/test/TimeOverTimeComparison_test.js

```diff
@@ -0,0 +1,150 @@
+// (C) 2007-2020 GoodData Corporation
+import { Selector } from "testcafe";
+import { loginUserAndNavigate, waitForPivotTableStopLoading, checkCellValue } from "./utils/helpers";
+import config from "./utils/config";
+
+const CELL_0_1 = ".s-cell-0-1";
+const CELL_0_2 = ".s-cell-0-2";
+const CELL_3_1 = ".s-cell-3-1";
+const CELL_3_2 = ".s-cell-3-2";
+
+const markers = ".highcharts-markers *";
+const dataLabels = ".highcharts-data-label tspan";
+const tooltipCssSelector = ".gd-viz-tooltip-item .gd-viz-tooltip-value";
+
+fixture("Time Over Time Comparison").beforeEach(
+    loginUserAndNavigate(`${config.url}/time-over-time-comparison`),
+);
+
+test("should compare the data with a comparing to the SPPY Column Chart", async t => {
```

Comment:
I would rename it to "should compare the data with SPPY in column chart" (so it flows better when reading it). Please update other tests on the page as well.